### PR TITLE
Require jittered o/d points to be a minimum distance apart

### PR DESCRIPTION
**Pull request by dabreegster**
For #1

### CODE REVIEW

**Comment by dabreegster** [comment]
@Robinlovelace, here ya go!

**Comment by Robinlovelace** [comment]
:fire:

File: src/lib.rs [hunk 1]
```diff
@@ -39,6 +40,8 @@ pub struct Options {
     pub origin_key: String,
     /// Which column in the OD row specifies the zone where trips ends?
     pub destination_key: String,
+    /// Guarantee that jittered points are at least this distance apart.
+    pub min_distance_meters: f64,
```
Comment (Robinlovelace) [inline src/lib.rs:44]: :+1: to explicit var (or R they arg?) names

File: src/lib.rs [hunk 2]
```diff
@@ -55,6 +58,10 @@ pub enum Subsample {
     UnweightedPoints(Vec<Point<f64>>),
 }
 
+/// TODO Describe what this does.
+///
+/// Note this assumes assumes all input is in the WGS84 coordinate system, and uses the Haversine
```
Comment (Robinlovelace) [inline src/lib.rs:63]: Fair as that's the GeoJSON spec.

**Review by Robinlovelace — APPROVED** [review]
No default value. Solid and puts the responsibility on the user IIRC. Amazing work, can test and report back if you want, saw lots of these in previous version: 

image

**Pull request merged**